### ISSUE TITLE
TensorFlowを試す

### ISSUE THREAD

**Issue by awakia** [awakia/catchup#68]

**Comment by awakia** [comment]
https://github.com/jupyter/docker-stacks/tree/master/tensorflow-notebook

```
docker run -it --rm -p 8888:8888 -v `pwd`:/home/jovyan/work jupyter/tensorflow-notebook

**Comment by awakia** [comment]
## keras でやってみるか

https://keras.io/ja/
https://qiita.com/yampy/items/706d44417c433e68db0d

**Comment by awakia** [comment]
- [ ] CNN
- [ ] Optimizer: Adam
- [ ] Batch Normalization
- [ ] Dropout

**Comment by awakia** [comment]
## RNN

penn treebank dataset

https://www.tensorflow.org/tutorials/recurrent
https://qiita.com/KojiOhki/items/149f96bd98973bd219ac

**Comment by awakia** [comment]
https://www.tensorflow.org/versions/r1.1/get_started/get_started

最近変わってしまったけど、昔のいいのでメモ
